src/components/jobs_view.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { ReactNode, useEffect, useState } from 'react';
+import { TfiTrash, TfiPlus, TfiPencil, TfiFilter } from "react-icons/tfi";
+import PouchDB from 'pouchdb'
+import PouchDBUpsert from 'pouchdb-upsert'
+import { Button, ListGroup } from 'react-bootstrap';
+import templatesConfig from '../templates/templates_config'
+import Dropdown from 'react-bootstrap/Dropdown';
+
+PouchDB.plugin(PouchDBUpsert);
+
+interface JobListProps {
+  dbName: string;
+}
+
+const JobList: React.FC<JobListProps> = ({ dbName }) => {
+  const db = new PouchDB(dbName);
+  const [sortedJobs, setSortedJobs] = useState<any[]>([]);
+
+  const retrieveJobs = async () => {
+    try {
+      const result = await db.allDocs({ include_docs: true });
+      //console.log(result.rows.map(row => row));
+      const sortedJobs = result.rows.map(row => row.doc?._id);
+      setSortedJobs(sortedJobs);
+    } catch (error) {
+      console.error('Error retrieving jobs:', error);
+    }
+  }
+
+    useEffect(() => {
+      retrieveJobs();
+    }, []);
+
+
+  const sortByCreateTime =  async () =>{
+    try {
+      const result = await db.allDocs({ include_docs: true });
```

Comment:
Do the sorting on the stored list in state instead of re-retrieving the list from the database.